app/models/incident_report.py

```diff
@@ -25,3 +26,34 @@ class IncidentReport(db.Model):
     agency_id = db.Column(db.Integer, db.ForeignKey('agencies.id'))
     picture_url = db.Column(db.Text)
     description = db.Column(db.Text)
+
+    @staticmethod
+    def generate_fake(count=100, **kwargs):
+        """Generate a number of fake reports for testing."""
+        from sqlalchemy.exc import IntegrityError
+        from random import seed, choice, randint
+        from datetime import timedelta
+        import forgery_py
+
+        agencies = Agency.query.all()
+
+        seed()
+        for i in range(count):
+            l = Location(original_user_text=forgery_py.address.street_address())
+            u = IncidentReport(
```

Comment:
very small/can ignore, but I would use r instead of u as the temp variable (short for 'report')